Automattic/wp-calypso
# Login: Add a backup codes screen

This pull request adds a new option to use backup codes for 2FA if the customer has lost their phone:
![screen_shot_2017-05-11_at_10 35 08](https://cloud.githubusercontent.com/assets/275961/26068088/3445512e-3994-11e7-8da6-1141c3683f0b.png)
![screen_shot_2017-05-11_at_10 37 41](https://cloud.githubusercontent.com/assets/275961/26068090/3612bc3a-3994-11e7-9c25-2c6c4f7a4599.png)
  
#### Testing instructions
  
1. Run `git checkout add/backup-codes` and start your server, or open a [live branch](https://calypso.live/?branch=add/backup-codes)
2. Open the [`Login` page](http://calypso.localhost:3000/login)
3. Enter a username and password
4. Check that on the 2FA screens you see an "I can't access my phone link" in the footer
5. Check that the "I can't access my phone" link takes you to a page to enter you backup code.
  
#### Reviews
  

   
@Automattic/sdev-feed
